modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -550,4 +553,71 @@ private function fieldFormatSocialElement($items, $body) {
       $body->addChild($social);
     }
   }
+
+  /**
+   * Add ads if configured in settings
+   */
+  public function addAdsFromSettings() {
+    $ad_type = variable_get('fb_instant_articles_ad_type', AdTypes::FB_INSTANT_ARTICLES_AD_TYPE_NONE);
+    if ($ad_type === AdTypes::FB_INSTANT_ARTICLES_AD_TYPE_NONE) {
+      return;
+    }
+    $dimensions = variable_get('fb_instant_articles_ads_dimension', AdTypes::FB_INSTANT_ARTICLES_AD_TYPE_NONE);
```

Comment:
I would say so, but you are absolutely right in [your other comment](https://github.com/BurdaMagazinOrg/module-fb_instant_articles/pull/36#discussion_r59301324) that this default should be applied there as well.